File: go/raindrops/raindrops.go
package raindrops

import (
	"strconv"
)

//Convert ...Convert numbers to Pling Plang Plong
func Convert(input int) string {

	output := ""

	if input%3 == 0 {
		output += "Pling"
	}
	if input%5 == 0 {
		output += "Plang"
	}
	if input%7 == 0 {
		output += "Plong"
	}

	if output != "" {
		return output
	}
	return strconv.Itoa(input)

}

File: julia/raindrops/raindrops.jl


function raindrops(number)
   
   raindrops = [(3,"Pling"),(5,"Plang"),(7,"Plong")]
   result  = map(x-> number % x[1] == 0 ? x[2] : "" ,raindrops) |> join
   result == "" ? string(number) : result
    
end
